pkgs/applications/misc/tessen/default.nix

```
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
```

```suggestion
{ lib
, stdenv
```
The common style is to put `lib` first.